.env.sample

```diff
@@ -1,2 +1,3 @@
 EDGI_ZOOM_API_KEY=
 EDGI_ZOOM_API_SECRET=
+NOTIFICATION_WEBHOOK_URL=https://my-hubot.herokuapp.com/youtube-upload-complete
```

Comment:
Given the example URL, this variable name seems overly generic and prone to confusion.